src/components/organisms/Dashboard/hooks.ts

```diff
@@ -145,7 +145,9 @@ export default (teamId?: string) => {
       .filter((project): project is Project => !!project);
   }, [team?.projects.nodes]);
 
-  const [createNewProject] = useCreateProjectMutation();
+  const [createNewProject] = useCreateProjectMutation({
+    refetchQueries: ["Project"],
```

Comment:
You have added this refetchQueries but in the createProject function we are still using refetch() as well.
I think we only need one. Also, if you are adding refetchQueries that probably means refetch() wasn't working properly, so either way it should be removed from the callback!